data/generateData.js

```diff
@@ -177,8 +181,11 @@ function getTopRoundsByScore(roundsByScore) {
   return topRounds;
 }
 
-function getStats(rounds) {
-  const statsByPlayer = Object.values(rounds.reduce(getStatsByPlayer, {}));
+function getStats(rounds, maxRoundsPerPlayer) {
```

Comment:
Add a default value here so you don't need to pass a value everywhere. Also, `MAX_SAFE_INTEGER` seems a bit excessive. At our current rate, we'd barely break 1000 rounds if we played consistently for the next 40 years.

```suggestion
function getStats(rounds, maxRoundsPerPlayer = Number.MAX_SAFE_INTEGER) {
```

Reply:
I personally think the default value is confusing. It means different call sites (only two) have different Params and means u have to look in more places to see what’s going on. But I’ll change it.

Reply:
Also, defensive programming! Otherwise 1,000 becomes a magic number that is pulled out of thin air.

Reply:
FWIW, I prefer the default value too. Don't feel strongly about 1000 vs. MAX_INT.

Reply:
Removed need for this in latest rev